File: CrystalEditor/Resources/Scripts/RotateObject.lua
function Start()
    transform:SetEulerRot(Maths.Vector3.new())
end

function LateStart()
end

function Update()
    transform:RotateEuler(Maths.Vector3.new(0, time:DeltaTime() * Maths.PI/8 , 0))
end

function LateUpdate()
end

-- CrystalEngine
-- 

File: CrystalEditor/Resources/Scripts/RotateObject.py
from CrystalPython import *

class RotateObject(ObjectScript):
    def Start(self):
        self.transform.SetEulerRot(Vector3())

    def Update(self):
        self.transform.RotateEuler(Vector3(0, self.time.DeltaTime() * PI/8 , 0))
